src/test/java/hw5/steps/AssertionSteps.java

```diff
@@ -0,0 +1,92 @@
+
+import java.util.List;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class AssertionSteps {
+
+    MainPage mainPage = new MainPage(TestContext.getInstance().getDriver());
+    ServicePage servicePage = new ServicePage(TestContext.getInstance().getDriver());
+    UserTablePage userTablePage = new UserTablePage(TestContext.getInstance().getDriver());
+
+    @Then("Browser title is equal to {string}")
+    public void browser_title_is_equal_to(String string) {
+        assertEquals(mainPage.getPageTitle(), string);
+    }
+
+    @Then("Correct username {string} is shown on the Main Page")
+    public void correct_username_is_shown_on_the_Main_Page(String string) {
+        assertEquals(mainPage.getPageUserName(), string);
+    }
+
+    @Then("Correct Water checkbox log is shown in the logs area")
+    public void correct_water_checkbox_log_is_shown_in_the_logs_area() {
+        assertTrue(servicePage.logContainsCheckboxWater());
+    }
+
+    @Then("Correct Wind checkbox log is shown in the logs area")
+    public void correct_wind_checkbox_log_is_shown_in_the_logs_area() {
+        assertTrue(servicePage.logContainsCheckboxWind());
+    }
+
+    @Then("Correct \"Selen\" radio button log is shown in the logs area")
+    public void correct_radio_button_log_is_shown_in_the_logs_area() {
+        assertTrue(servicePage.logContainsRadiobuttonSelen());
+    }
+
+    @Then("Correct dropdown menu log for \"Yellow\" color is shown in the logs area")
+    public void correct_dropdown_menu_log_for_color_is_shown_in_the_logs_area() {
+        assertTrue(servicePage.logContainsSelectedColor());
+    }
+
+    @Then("{string} page should be opened")
+    public void page_should_be_opened(String string) {
+        assertEquals(userTablePage.getPageTitle(), string);
+    }
+
+    @Then("{int} Number Type Dropdowns should be displayed on Users Table on User Table Page")
+    public void number_Type_Dropdowns_should_be_displayed_on_Users_Table_on_User_Table_Page(Integer int1) {
+        assertEquals((Integer) userTablePage.getDropDownList().size(), int1);
+    }
+
+    @Then("{int} Usernames should be displayed on Users Table on User Table Page")
+    public void usernames_should_be_displayed_on_Users_Table_on_User_Table_Page(Integer int1) {
+        assertEquals((Integer) userTablePage.getUserNameList().size(), int1);
+    }
+
+    @Then("{int} Description texts under images should be displayed on Users Table on User Table Page")
+    public void description_texts_under_images_should_be_displayed_on_Users_Table_on_User_Table_Page(Integer int1) {
+        assertEquals((Integer) userTablePage.getDescriptionList().size(), int1);
+    }
+
+    @Then("{int} checkboxes should be displayed on Users Table on User Table Page")
+    public void checkboxes_should_be_displayed_on_Users_Table_on_User_Table_Page(Integer int1) {
+        assertEquals((Integer) userTablePage.getCheckboxList().size(), int1);
+    }
```

Comment:
Please provide more clear names for the method input parameters

Reply:
Done.